src/main/resources/static/css/image-upload.css

```
@@ -0,0 +1,162 @@
+    margin-top: 8px;
+}
+
+/* 파일 input 은 감추되 포커스는 받을 수 있게 남겨둔다 (display:none 이면 못 받는다) */
+.image-upload-input {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)